cmd/tools/users/root.go

```diff
@@ -0,0 +1,18 @@
+package users
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var RootCMD = &cobra.Command{
+	Use:   "users",
+	Short: "Platform tools for users",
+	Long: `
+
+Tools to interact with the users in the platform`,
+}
+
+func init() {
+	RootCMD.AddCommand(addUserToCustomerCMD)
+	RootCMD.AddCommand(listEmailsCMD)
```

Comment:
```suggestion
	RootCMD.AddCommand(listEmailsCMD)
	RootCMD.AddCommand(getUserCMD)
```

The `get-user` command wasn't available as it wasn't part of the root command.